met-web/src/components/common/Modals/Schedule/index.tsx

```diff
@@ -39,7 +53,7 @@ const ScheduleModal = ({ reschedule, open, updateModal }: ScheduleModalProps) =>
             );
             return;
         }
-        if (scheduledDate)
+        if (scheduledDate && validateDate())
```

Comment:
You can simplify this if with just the validate method, and move the date check to to Validate one

Reply:
done :)